.github/workflows/build.yml

```diff
@@ -23,7 +23,7 @@ jobs:
     php: [ 7.4, 8.0 ]
     node: [ 10.x ]
     symfony: [ ^4.4, ^5.2 ]
-    sylius: [ ^1.10.0 ]
+    sylius: [ ^1.8.0, ^1.9.0, ^1.10.0 ]
```

Comment:
All tests will run on the latest Sylius version - 1.10.1

Reply:
and 1.9
```
            matrix:
                php: ["8.0", "7.4"]
                symfony: ["^4.4", "^5.2"]
                sylius: ["~1.9.0", "~1.10.0"]
                node: ["10.x"]
                mysql: ["8.0"]

                exclude:
                    - sylius: ~1.9.0
                      php: 8.0

                    - sylius: ~1.10.0
                      symfony: 4.4
```